lab-davide/model/note.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const noteSchema = mongoose.Schema({
+  name: {type: String, required: true},
+  timestamp: {type: Date, required: true},
```

Comment:
Instead of building this in your route and having required:true, you can do something like: 
```javascript
  timestamp: { type: Date, default: Date.now },
```
That  does the same thing but saves you some time. 

You can also implement mongoose's built in support for [createdAt and updatedAt timestamps](http://mongoosejs.com/docs/guide.html#timestamps) as a schema option. In this case, you would do it like:

```javascript 
const noteSchema = mongoose.Schema({
  name:  { type: String, required: true },
  content: { type: String, required: true }
}, {
  timestamps: {
    createdAt: 'timestamp'
  }
});
```
That also does the same thing.